xxh_xxh/xxh.py

```diff
@@ -70,6 +70,13 @@ def eprint(self, *args, **kwargs):
     def eeprint(self, *args, **kwargs):
         eeprint(*args, **kwargs)
 
+    @staticmethod
+    def generate_config_filepath():
+        if os.environ.get('XDG_CONFIG_HOME'):
+            return p(os.environ['XDG_CONFIG_HOME']) / 'xxh' / 'xxhc'
+        else:
+            return p(os.environ['HOME']) / 'xxh' / 'xxhc'
```

Comment:
this should default to `~/.config` if it is to follow the [XDG spec](https://wiki.archlinux.org/index.php/XDG_Base_Directory).

Reply:
$HOME should be OK. From the actual spec:

> $XDG_CONFIG_HOME defines the base directory relative to which user specific configuration files should be stored. If $XDG_CONFIG_HOME is either not set or empty, a default equal to $HOME/.config should be used.

Reply:
... which states that the default should be `~/.config`, not `~`

Reply:
@snowe2010 Ah, I didn't notice I had messed up the code. Cheers